build_rom.sh

```diff
@@ -1,12 +1,12 @@
 # sync rom
-repo init --depth=1 -u git://github.com/AospExtended/manifest.git -b 11.x -g default,-device,-mips,-darwin,-notdefault
-git clone https://github.com/Apon77Lab/android_.repo_local_manifests.git --depth 1 -b aex .repo/local_manifests
+repo init --depth=1 -u https://github.com/StagOS/manifest.git -b r11 -g default,-device,-mips,-darwin,-notdefault
+git clone https://github.com/kardebayan/local_manifests.git --depth 1 -b r11 .repo/local_manifests
 repo sync -c --no-clone-bundle --no-tags --optimized-fetch --prune --force-sync -j8 || repo sync -c --no-clone-bundle --no-tags --optimized-fetch --prune --force-sync -j8
 
 # build rom
 source build/envsetup.sh
-lunch aosp_mido-user
-m aex
+lunch stag_CPH1859-eng
+make installclean && make stag
```

Comment:
you dont need installclean, server already does installclean by default

Reply:
Oh, Thanks for informing